src/Component.cpp

```diff
@@ -3,12 +3,12 @@
 */
 #include "Arduino.h"
 #include "Component.h"
+#include "Logger.h"
 
 Component::Component(int pin, String componentType){
-  if(Properties::DEBUGGING_ACTIVE) Serial.println("Component " + componentType + " is set at pin " + pin);
+  Logger::Log("Component " + componentType + " is set at pin " + pin);
   _pin = pin;
   _componentType = componentType;
-  delay(100);
```

Comment:
This was the "fix" we implemented before for that 'race condition' we were having. Did another fix take care of that so this small delay is no longer needed? What fix was that?